kiota_abstractions/request_information.py

```diff
@@ -227,13 +227,17 @@ def set_content_from_scalar(
                 writer_func(None, values)
             self._set_content_and_content_type_header(writer, content_type)
 
-    def set_stream_content(self, value: BytesIO) -> None:
+    def set_stream_content(self, value: BytesIO, content_type: Optional[str]) -> None:
```

Comment:
Might need to default to binary or minor rev

Reply:
so that's where my lack of experience with python shows.
I'm defaulting the value below. And I thought Optional[str] meant that one could call the method without passing the content type argument?

Reply:
```suggestion
    def set_stream_content(self, value: BytesIO, content_type: str = None) -> None:
```